source/api/presentation/3.0/index.md

```diff
@@ -265,6 +265,57 @@ The value _MUST_ be a string. If the value is drawn from Creative Commons or Rig
 { "rights": "https://creativecommons.org/licenses/by/4.0/" }
 ```
 
+##### provider
+
+An organization or person that contributed to providing the content of the resource. Clients can then display this information to the user to acknowledge the provider's contributions.  This differs from the `requiredStatement` property, in that the data is structured, allowing the client to do more than just present text but instead have richer information about the people and organizations to use in different interfaces.
```

Comment:
should "can then" be _MAY_ ?

Reply:
No, it's SHOULD in the normative part for Collection and Manifest.  "can" was intentionally chosen to NOT seem like an RFC word.

Reply:
I guess I'm stuck with the old fashioned British permission/ability distinction between may/can ;-) I don't see a simple change so I'll :+1: